### PR TITLE
Add ability to override number of reducers on group by entity job

### PR DESCRIPTION
**Pull request by raronson**
@markhibberd @charleso

### CODE REVIEW

File: ivory-operation/src/main/scala/com/ambiata/ivory/operation/extraction/output/GroupByEntityOutput.scala [hunk 1]
```diff
@@ -5,15 +5,19 @@ import com.ambiata.ivory.storage.lookup.ReducerSize
 import com.ambiata.mundane.control._
 import com.ambiata.mundane.io.MemoryConversions._
 
-import scalaz.effect.IO
+import scalaz._, Scalaz._, effect.IO, \&/._
 
 object GroupByEntityOutput {
+  val GroupByEntityNumReducers = "GroupByEntity.reducers"
```
Comment (markhibberd) [inline ivory-operation/src/main/scala/com/ambiata/ivory/operation/extraction/output/GroupByEntityOutput.scala]: If it is ok, can you prefix it with `ivory.`

**Comment by charleso** [comment]
:+1:

**Comment by raronson** [comment]
@charleso i pushed the property to the edge and passed it in

**Comment by charleso** [comment]
:+1:

**Pull request merged**